frontend/packages/console-shared/src/components/shortcuts/Shortcut.tsx

```diff
@@ -15,14 +15,17 @@ interface ShortcutProps {
   keyName?: string;
   rightClick?: boolean;
   shift?: boolean;
+  toggleTabAction?: boolean;
 }
 
-const Command: React.FC = ({ children }) => (
+export const ShortcutCommand: React.FC = ({ children }) => (
   <span className="ocs-shortcut__command">
     <kbd>{children}</kbd>
   </span>
 );
 
+export const isMac = () => window.navigator.platform.includes('Mac');
```

Comment:
You're using `isMac` like it is a constant value although you have defined it as a function. You'll need to evaluate this in place to continue to use it as a constant:

```suggestion
export const isMac = window.navigator.platform.includes('Mac');
```

Reply:
updated